dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -19,45 +19,41 @@
 
-JAVAEESEC_ERROR_GEN_DB=CWWKS1918E: Cannot validate the credentials for caller {0}. The DatabaseIdentityStore failed to execute the query ''{1}'' with an error: {2}
-JAVAEESEC_ERROR_GEN_DB.explanation=The search for the caller failed with an error.
-JAVAEESEC_ERROR_GEN_DB.useraction=Review the error message.
+JAVAEESEC_ERROR_GEN_DB=CWWKS1918E: The credentials for the {0} caller cannot be validated. The DatabaseIdentityStore object failed to run the ''{1}'' query with an error: {2}
+JAVAEESEC_ERROR_GEN_DB.explanation=The search for the caller failed with an error. <= Give additional information to explain why it wasn't found, maybe some examples of what the problem could be. <<< There are various cause why thie error is reported. A user need to review errors in a message.log file.
+JAVAEESEC_ERROR_GEN_DB.useraction=Review the error message and make corrections to the search based on the error message.
 
-JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS=CWWKS1919W: The DatabaseIdentityStore failed to execute the query ''{1}''  to get the groups for the caller {0}. The partial list of groups is {2}. The error is {3}.
-JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS.explanation=The search for groups for the caller failed with an error.
-JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS.useraction=Review the provided error. A partial list of groups is returned.
+JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS=CWWKS1919W: The DatabaseIdentityStore failed to run the ''{1}'' query to get the groups for the {0} caller. The partial list of groups is {2}. The error is {3}.
+JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS.explanation=A partial list of groups for the caller is returned instead of a complete list of groups because of an error in the query.
+JAVAEESEC_WARNING_EXCEPTION_ON_GROUPS.useraction=Review the provided error and make corrections to the query so that the complete list of groups for the caller can be returned. 
 
-JAVAEESEC_WARNING_WRONG_CRED=CWWKS1920W: The credential provided to the IdentityStore is not a UsernamePasswordCredential and cannot be validated.
-JAVAEESEC_WARNING_WRONG_CRED.explanation=The credential provided to the IdentityStore is not a UsernamePasswordCredential and cannot be validated.
-JAVAEESEC_WARNING_WRONG_CRED.useraction=Review the type of credential passed into the IdentityStore.
-
-JAVAEESEC_WARNING_NULL_PWD=CWWKS1921W: A null password was provided to the DatabaseIdentityStore and cannot be validated.
-JAVAEESEC_WARNING_NULL_PWD.explanation=The UsernamePasswordCredential contained a null password.
-JAVAEESEC_WARNING_NULL_PWD.useraction=Supply a valid password.
+JAVAEESEC_ERROR_WRONG_CRED=CWWKS1920E: The credential provided to the IdentityStore is not a UsernamePasswordCredential and cannot be validated.
+JAVAEESEC_ERROR_WRONG_CRED.explanation=The credential provided to the IdentityStore is not a UsernamePasswordCredential and cannot be validated.
```

Comment:
Put a noun after IdentityStore and UsernamePasswordCredential. Doing so provides context for users and translators, i.e. IdentityStore object and UsernamePasswordCredential _blah_

Reply:
Done.